tap_teamwork/__init__.py

```diff
@@ -22,22 +27,21 @@ def do_discover():
 @singer.utils.handle_top_exception(LOGGER)
 def main():
     """
-    Run the tap
+    Parse command-line arguments and run discovery or sync mode.
     """
     parsed_args = singer.utils.parse_args(REQUIRED_CONFIG_KEYS)
-    state = {}
-    if parsed_args.state:
-        state = parsed_args.state
+    state = parsed_args.state or {}
 
     with Client(parsed_args.config) as client:
         if parsed_args.discover:
             do_discover()
         elif parsed_args.catalog:
             sync(
-                client=client,
-                config=parsed_args.config,
-                catalog=parsed_args.catalog,
-                state=state)
+                client,
+                parsed_args.config,
+                parsed_args.catalog,
+                state
+            )
```

Comment:
Why was this required? I this looked clear before, as it improves readability and reduces the risk of bugs caused by misordered arguments.